package com.mega.mvc06;

import java.io.FileWriter;
import java.io.IOException;

public class Problem05Main {

	public static void main(String[] args) {
		Problem05 error = new Problem05();
		error.call();
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
		System.out.println("에러 발생후 실행될까요....!!!!s");
	}

}
